analysis_runner/dataproc.py

```diff
@@ -78,6 +78,8 @@ def hail_dataproc_job(
     main_job.command(f'git clone {git_remote} {repo_name}')
     main_job.command(f'cd {repo_name}')
     main_job.command(f'git checkout {git_sha}')
+    main_job.command(f'git submodule init')
+    main_job.command(f'git submodule update')
```

Comment:
```suggestion
    main_job.command(f'git clone --recurse-submodules {git_remote} {repo_name}')
    main_job.command(f'cd {repo_name}')
    main_job.command(f'git checkout {git_sha}')
```

Reply:
I don't think it would work if the main branch doesn't yet have the submodules?

Reply:
Still need `git submodule update` after checkout